ktor-io/jvm/src/io/ktor/utils/io/ByteChannelSequentialJVM.kt

```diff
@@ -86,10 +86,6 @@ public class ByteChannelSequentialJVM(
     }
 
     override fun readAvailable(min: Int, block: (ByteBuffer) -> Unit): Int {
-        if (closed) {
-            throw closedCause ?: ClosedSendChannelException("Channel closed for read")
```

Comment:
It looks like we should throw `closedCause` if it's not null

Reply:
thanks, don't know why I lost it. fixed!